packages/endo/bin/endo.js

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env node
-import fs from "fs";
-import { main } from "../src/cli.js";
-
-main(process, { fs: fs.promises });
+(async () => {
+  const fs = await import("fs");
+  const { main } = await import("../src/cli.js");
+  main(process, { fs: fs.promises });
+})();
```

Comment:
This had to become a CommonJS module. Node.js doesn’t yet recognize shell scripts as ESM when they are ESM per the containing package.json. This might be for lack of canonicalizing the script location.